eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/Handler.php

```diff
@@ -683,6 +683,6 @@ protected function removeSubtree($id, $action, $original)
      */
     protected function getHash($text)
     {
-        return md5(strtolower($text));
+        return md5(mb_strtolower($text));
```

Comment:
@yannickroger is right. Default internal encoding for Mbstring is ISO-8859-1 in PHP 5.3. IMHO It's OK to leave this as is for master and 5.4, but we need to specify UTF-8 for 5.3.